src/components/Modeler.vue

```diff
@@ -602,22 +610,16 @@ export default {
       return new Promise(resolve => setTimeout(resolve, cursorWaitTime));
     },
     async renderPaper() {
-      await this.$nextTick();
```

Comment:
A call to `$nextTick` was already being made in `waitForCursorToChange`, so calling it here seemed redundant.